articles/quickstart/native/android/07-authorization.md

```diff
@@ -21,40 +21,17 @@ This tutorial shows you how to use Auth0 to create access roles for your users.
 Be sure that you have completed the [Login](/quickstart/native/android/00-login) quickstart.
 :::
 
-Create a rule that assigns the users either an `admin` role, or a simple `user` role. Go to the [new rule page](${manage_url}/#/rules/new) and select the "Set Roles To A User" template, under **Access Control**. Replace the default script contents with the following snippet:
+Create a [Rule](https://auth0.com/docs/rules) that assigns the users either an `admin` role, or a simple `user` role. Go to the [new rule page](${manage_url}/#/rules/new) and select the **Set Roles To A User** template, under **Access Control**. Edit the following lines from the default script to match the conditions that fit your needs:
 
 ```js
-function (user, context, callback) {
```

Comment:
this line is the signature of a typical Auth0 Rule. If you remove it, how does the user know they need to put the code below "inside" the default rule code ?

Reply:
The user does not need to put the code inside the Rule, they just have to search for the relevant lines and change them as needed. Hence the (new) instruction above reads: `Edit the following lines from the default script to match the conditions that fit your needs`.